helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -174,3 +222,18 @@ service:
         {{- end }}
     {{- end }}
 {{- end }}
+
+{{/*
+Pod anti-affinity to prevent eks/fargate replicas from being on same node
+*/}}
+{{- define "splunk-otel-collector.clusterReceiverPodAntiAffinity" -}}
```

Comment:
@dmitryax is this an ok location for the anti-affinity default? I wonder if the convention is to have it in common, though it's not used in any other component.

Reply:
https://github.com/signalfx/splunk-otel-collector-chart/pull/346#issuecomment-1026962298

Reply:
actually looks like helm 3.5.4 doesn't like this, so I'm not sure of the "correct" way that would satisfy your objection: https://github.com/signalfx/splunk-otel-collector-chart/runs/5023506147?.

Reply:
err, backtick on the variable definition action resolves...